spec/views/locations/index.html.jbuilder_spec.rb

```diff
@@ -1,11 +1,10 @@
 require 'rails_helper'
```

Comment:
Am I missing something or is this file badly named? By my reckoning it shouldn't have the `.html` part...